aw.spec

```
@@ -245,14 +245,12 @@ if platform.system() == "Darwin":
         name="ActivityWatch.app",
         icon=icon,
         bundle_identifier="net.activitywatch.ActivityWatch",
-        # Not respected?
-        codesign_identity=codesign_identity,
-        entitlements_file=entitlements_file,
         version=current_release.lstrip('v'),
         info_plist={
             "NSPrincipalClass": "NSApplication",
             "CFBundleExecutable": "MacOS/aw-qt",
-            "CFBundleIconFile": "logo.icns",
+            "CFBundlpaseIconFile": "logo.icns",
```

I assume this is a typo?

```suggestion
            "CFBundleIconFile": "logo.icns",
```